OpenKeychain/src/main/java/org/sufficientlysecure/keychain/ui/ViewKeyAdvShareFragment.java

```diff
@@ -380,6 +413,24 @@ public void onLoadFinished(Loader<Cursor> loader, Cursor data) {
 
         }
         setContentShown(true);
+        showSshShareIfApplicable();
+    }
+
+    private void showSshShareIfApplicable() {
```

Comment:
@dschuermann The SSH export option only appears if there is a authentication key in the selected keyring

Reply:
great

Reply:
Please show it regardless of auth key, and display an error if there is none. An option that's just missing for complicated reasons breaks user expectations in a very frustrating way.

Reply:
Ok, that makes sense. I was concerned it might clutter up the UI with a feature that will not be used by the majority of the users. I have changed it to display a Notify instead.